addons/flares/CfgAmmo.hpp

```diff
@@ -0,0 +1,14 @@
+class CfgAmmo {
+    class CUP_F_40mm_Star_White;
+    class CUP_F_40mm_StarCluster_White: CUP_F_40mm_Star_White {
+        timeToLive = 60;
```

Comment:
Do vanilla ones do 60 or less? Should be same as GL rounds.

Reply:
The issue is it has a regular ammo class & submunition, the one that isn't used is set really high, the one that IS used is set really low which makes them totally useless. Fire it and it's gone before you've reloaded.